packages/core/strapi/lib/core/registries/custom-fields.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { has } = require('lodash/fp');
+
+const customFieldsRegistry = () => {
+  const customFields = {};
+
+  return {
+    getAll() {
+      return customFields;
+    },
+    register(customField) {
```

Comment:
it's too bad it doesn't match with the other registries. 

I think we probably want to split the user API vs the internal registry.